packages/utilities/src/createRef.ts

```diff
@@ -0,0 +1,13 @@
+export type RefObject<T> = {
+  (component: T): void;
+  value: T | null;
+};
+
+export function createRef<T>(): RefObject<T> {
+  const refObject = ((element: T): void => {
```

Comment:
I *think* `(element: T | null): void` is the right signature, but i don't think it matters.

Reply:
I *think* you are right :P